desc/magnetic_fields/_current_potential.py

```diff
@@ -842,3 +1031,878 @@ def nfp_loop(j, f):
+    fields::
+
+        Bn = ∫ ∫ (B . n)^2 dA
+        B = B_plasma + B_external + B_Phi_SV + B_GI
+
+    G is fixed by the equilibrium magnetic field strength, and I is determined
+    by the desired coil topology (given by ``current_helicity``), with zero
+    helicity corresponding to modular coils, and non-zero helicity corresponding
+    to helical coils. The algorithm then finds the single-valued part of Φ
+    by minimizing the quadratic flux on the plasma surface along with a
+    regularization term on the surface current magnitude::
+
+        min_Φₛᵥ  ∫ ∫ (B . n)^2 dA + λ ∫ ∫ | K |^2 dA
+
+    where λ is the regularization parameter, smaller `lambda_regularization`
+    corresponds to less regularization (consequently, lower Bn error but more
+    complex and large surface currents) and larger `lambda_regularization`
+    corresponds to more regularization (consequently, higher Bn error but simpler
+    and smaller surface currents).
+
+    [1] Landreman, An improved current potential method for fast computation
+        of stellarator coil shapes, Nuclear Fusion (2017)
+
+    Parameters
+    ----------
+    field : FourierCurrentPotentialField
+        ``FourierCurrentPotentialField`` to run REGCOIL with.
+    eq : Equilibrium
+        Equilibrium to minimize the quadratic flux (plus regularization) on.
+    lambda_regularization : float or ndarray, optional
+        regularization parameter, >= 0, regularizes minimization of Bn
+        on plasma surface with minimization of current density mag K on winding
+        surface i.e. larger lambda_regularization, simpler coilset and smaller
+        currents, but worse Bn. If a float, only runs REGCOIL for that single value
+        and returns a list with the single FourierCurrentPotentialField and the
+        associated data.
+        If an array is passed, will run REGCOIL for each lambda_regularization in
+        that array and return a list of FourierCurrentPotentialFields, and the
+        associated data.
+    current_helicity : tuple of size 2, optional
+        Tuple of (q,p) used to determine coil topology, where q is the
+        number of poloidal transits a coil makes in one field period and
+        p is the number of toroidal transits a coil makes in one field period.
+        if p is zero and q nonzero, it corresponds to modular coil topology.
+        If both p,q are nonzero, it corresponds to helical coils.
+        If p,q are both zero, it corresponds to windowpane coils.
+        The net toroidal current (when q is nonzero) is set as
+        I = p(G-G_ext)/q/NFP
+    vacuum : bool, optional
+        if True, will not include the contribution to the normal field from the
+        plasma currents.
+    regularization_type : {"simple","regcoil"}
+        whether to use a simple regularization based off of just the single-valued
+        part of Phi, or to use the full REGCOIL regularization penalizing | K | ^ 2.
```

Comment:
I think that a space is also ok? but will see after fixing the reference errors

Reply:
Should mention that the default is `"regcoil"`

Reply:
See comment above, it would help to clarify what is actually being penalized with the `"simple"` option

Reply:
Because in the constructor I use "regcoil" as the default, I don't think usually it also gets mentioned in the docstring. I can add tho

And yes I checked it looks fine as it is in the docs